package.json

```diff
@@ -30,6 +30,7 @@
     "db:use": "node ./scripts/db-use.js",
     "elastic:activate": "node ./scripts/elastic.js activate",
     "elastic:deactivate": "node ./scripts/elastic.js deactivate",
+    "elastic:hq": "node ./scripts/elastic.js hq",
```

Comment:
As we remove elastic hq, we need to remove this line as well. right?

Reply:
:grimacing: good point !